src/layerTree.js

```diff
@@ -104,8 +104,7 @@ const getFlattenLayers = (layersTree = []) => {
  */
 const getLayerZindex = (layersTree, layerTemplateKey) => {
     const flattenLayers = getFlattenLayerTree(layersTree);
-    const zIndex = flattenLayers.findIndex((l) => l.key === layerTemplateKey);
-    return zIndex || 0;
+    return flattenLayers.findIndex((l) => l.key === layerTemplateKey);
```

Comment:
AFAIK we don't want -1 as z-index. What do you think @vdubr?

Reply:
Z-index -1 is only when no layerTemplateKey will fit. Question is if return value should be -1 or null.

Reply:
just making clear it was working like that previously too:
```js
 -1 || 0 // => -1 (https://developer.mozilla.org/en-US/docs/Glossary/Falsy)
```

so the removed check was redundat as only case that right side of `zIndex || 0` would be used was if zIndex was already `0`.

Reply:
> Z-index -1 is only when no layerTemplateKey will fit. Question is if return value should be -1 or null.

What are valid values for zIndex?

Reply:
@vdubr (or whoever wrote this originally) When is the template key not found? What is the return value used for?
Return value -1 doesn't make sense to me, but maybe it's ok. Maybe null is better. Or maybe an error should be thrown.

Reply:
This function is used only once there: https://github.com/gisat/panther-front-office/blob/master/src/state/Maps/actions.js#L141 . Current usage knows that layerkey exists, so no check is needed. 

It would be better to return null instead of -1.

Reply:
fixed. `null` is returned instead of `-1`